internal/ethapi/api.coreth.go

```diff
@@ -0,0 +1,118 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ethapi
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/coreth/params"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/common/math"
+)
+
+const (
+	nAVAX = 1_000_000_000
+
+	minBaseFee       = params.EtnaMinBaseFee // 1 nAVAX
+	maxNormalBaseFee = 100 * nAVAX
+
+	minGasTip       = 1 // 1 wei
+	maxNormalGasTip = 20 * nAVAX
+
+	slowFeeNumerator = 19 // 19/20 = 0.95
+	fastFeeNumerator = 21 // 21/20 = 1.05
+	feeDenominator   = 20
+)
+
+var (
+	bigMinBaseFee       = big.NewInt(minBaseFee)
+	bigMaxNormalBaseFee = big.NewInt(maxNormalBaseFee)
+
+	bigMinGasTip       = big.NewInt(minGasTip)
+	bigMaxNormalGasTip = big.NewInt(maxNormalGasTip)
+
+	bigSlowFeeNumerator = big.NewInt(slowFeeNumerator)
+	bigFastFeeNumerator = big.NewInt(fastFeeNumerator)
+	bigFeeDenominator   = big.NewInt(feeDenominator)
+)
+
+type Price struct {
```

Comment:
nit I'm not too sure what's meant to be exported or not given this is PR in a series of PRs I suppose 😉 
Maybe to avoid having exported and unused types (especially in the internal directory which isn't exposed to other repositories), have everything unexported and export-as-needed in subsequent PRs?